bluemira/base/constants.py

```diff
@@ -42,68 +59,192 @@
 ONE_4PI = 1 / (4 * np.pi)
 
 # Gravitational constant
-GRAVITY = 9.81  # [m/s^2]  # nO ESCAPING
+GRAVITY = ureg.Quantity("gravity").to_base_units().magnitude  # [m/s^2]  # nO ESCAPING
 
-# Avogadro's number
-N_AVOGADRO = 6.02214e23  # [1/mol] Number of particles in a mol
+# Avogadro's number, [1/mol] Number of particles in a mol
+N_AVOGADRO = ureg.Quantity("avogadro_number").to_base_units().magnitude
 
 # Stefan-Boltzmann constant: black-body radiation constant of proportionality
-SIGMA_BOLTZMANN = 5.670367e-8  # [W/m^2.K^4]
+SIGMA_BOLTZMANN = ureg.Quantity("sigma").to_base_units().magnitude  # [W/m^2.K^4]
 
 # Boltzmann constant kB = R/N_a
-K_BOLTZMANN = 1.38064852e-23  # [J/K]
+K_BOLTZMANN = ureg.Quantity("k_B").to_base_units().magnitude  # [J/K]
 
-# Tritium half-life
-T_HALFLIFE = 12.32  # [yr]
+# neutron molar mass, [u] or [g/mol]
+NEUTRON_MOLAR_MASS = (
+    ureg.Quantity("m_n").to("g") * ureg.Quantity("avogadro_constant").to_base_units()
+).magnitude
 
-# Tritium decay constant
-T_LAMBDA = np.log(2) / T_HALFLIFE  # [1/yr]
+# proton molar mass, [u] or [g/mol]
+PROTON_MOLAR_MASS = (
+    ureg.Quantity("m_p").to("g") * ureg.Quantity("avogadro_constant").to_base_units()
+).magnitude
 
-# Tritium molar mass
-T_MOLAR_MASS = 3.016050  # [u] or [g/mol]
+# electron molar mass, [u] or [g/mol]
+ELECTRON_MOLAR_MASS = (
+    ureg.Quantity("m_e").to("g") * ureg.Quantity("avogadro_constant").to_base_units()
+).magnitude
 
-# Deuterium molar mass
-D_MOLAR_MASS = 2.014102  # [u] or [g/mol]
+# Tritium half-life
+# https://www.nist.gov/pml/radiation-physics/radioactivity/radionuclide-half-life-measurements
+# http://www.lnhb.fr/nuclear-data/nuclear-data-table/
+# http://www.lnhb.fr/nuclides/H-3_tables.pdf
+T_HALFLIFE = 12.312  # [yr]
```

Comment:
Ah ha! Interesting... this will change results, although I don't think we have any proper regression tests on the fuel cycle model at present.